Added portscanner.py

Corbe30 opened the pull request:
Relies on python3, might need to comment out the first line because windows powershell throws an error.

Corbe30 committed:
Added portscanner.py

Relies on python3, might need to comment out the first line because windows powershell throws an error.

sickwiz commented:
hello @Corbe30 .  This will need few changes before we can merge it into master:
i) Typo on line 31. Change "resolver" to "resolved".
ii) Implement threading to increase the speed of port scans. Without threading, you won't be able to scan external IPs i.e it will take a lot of time.
iii) Increase the number of ports.

sickwiz commented:
References :
[threading](https://www.pythontutorial.net/advanced-python/python-threading/)
[ports](https://github.com/Repozo/Web-Vulnerability-Scanner/blob/master/commonPorts.py)

Corbe30 committed:
Updated portScanner.py

Added threading and a longer list of ports that is going to be referenced from another py file.

Corbe30 committed:
Added Common port list

This list is a much longer list than the previous list which was directly stored in the main file.

sickwiz commented:
Merged. Good work.
